cpp/src/Ice/Network.h

```diff
@@ -33,7 +33,7 @@ typedef int ssize_t;
 #    include <unistd.h>
 #endif
 
-#if defined(__linux__) && !defined(ICE_NO_EPOLL)
+#if defined(__linux__)
 #    define ICE_USE_EPOLL 1
 #elif (defined(__APPLE__) || defined(__FreeBSD__) || defined(__FreeBSD_kernel__)) && TARGET_OS_IPHONE == 0
```

Comment:
(Unrelated to this PR)
I don't see how this compile on FreeBSD: TARGET_OS_IPHONE is defined on Apple, not FreeBSD.